src/App.js

```diff
@@ -102,6 +121,7 @@ class App extends Component {
     return classes;
   };  // End propsToClasses()
 
+  /** Render the App component */
```

Comment:
(Current discussion pending - is this description of React behavior a useful one?)

Reply:
So far we've decided that describing React behavior isn't something we want to actively pursue, but that hinting at it in the documentation may be useful. For example, in component classes and functions we may add an `@returns React.node` or `element` or something like that. Not `component` - apparently that's inaccurate because the use of them gets you an instance, not the class itself.